components/SettingsControls/src/SettingsCard/SettingsCardAutomationPeer.cs

```diff
@@ -7,7 +7,7 @@ namespace CommunityToolkit.WinUI.Controls;
 /// <summary>
 /// AutomationPeer for SettingsCard
 /// </summary>
-public partial class SettingsCardAutomationPeer : FrameworkElementAutomationPeer
+public partial class SettingsCardAutomationPeer : ButtonBaseAutomationPeer
```

Comment:
@Jaylyn-Barbee will Narrator announce this control now as "Button" because of the change in AutomationPeer? I don't think so, right? (As that's set by the type?

Reply:
It actually does say "A clickable settings card - Button".

Reply:
Interesting, I don't know what the guidance here is for this. We can see if we get any other feedback from the change.